README.md

```diff
@@ -35,43 +35,106 @@ to your Github account:
 3. Go to: https://github.com/settings/notifications
 4. Under "Subscriptions.. Participating, @mentions and custom", ensure that you have "Email" ticked
 
-You should now receive an email approximately every two weeks containing the changes made to the APIs since the last update.
+You should now receive an email approximately every two weeks containing the changes made to the APIs since the last
+update.
 
 * Please note that the date shown is the date that the change was deployed to the Sandbox or Production.
 
 ---
 
+### xx September 2023
+
+The following changes are available in sandbox:
+
+#### individuals-income-received-api
+
+* New `STATEFUL` Gov-Test-Scenarios were added to the following v2.0 endpoints:
+    * `Create and Amend`, `Retrieve` and `Delete Other Income`
+    * `Create and Amend`, `Retrieve` and `Delete Savings Income`
+    * `Create and Amend`, `Retrieve` and `Delete Other Capital Gains and Disposals`
+
+#### individuals-state-benefits-api
+
+* New `STATEFUL` Gov-Test-Scenarios were added to the following v1.0 endpoints:
+    * `Create`, `Amend`, `List` and `Delete State Benefit`
+    * `Amend` and `Delete State Benefit Amounts`
+
+#### self-assessment-accounts-api
+
+* New `STATEFUL` Gov-Test-Scenarios were added to the following v2.0 endpoints:
+    * `Create or Amend`, `Retrieve` and `Delete Coding Out Underpayments and Debt Amounts`
+
+#### cis-deductions-api
+
+* New `STATEFUL` Gov-Test-Scenarios were added to the following v2.0 endpoints:
+    * `Create` `Amend`, `Retrieve` and `Delete CIS Deductions for Subcontractor`
+
+#### self-assessment-bsas-api
+
+* New dynamic Gov-Test-Scenarios were added to the following v4.0 endpoints:
```

Comment:
```suggestion
* New `DYNAMIC` Gov-Test-Scenarios were added to the following v4.0 endpoints:
```

Reply:
I did think about this, but then realised the gov test scenarios have the word `..DYNAMIC...` somewhere in them, but are not called `DYNAMIC`. Do you still think `DYNAMIC` is the best descriptor to use?

Reply:
Hmm that makes sense. Leave it as it is 👍